scripts/authors.scala

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+exec scala "$0" "$@"
+!#
+
+/*
+ * Usage:
+ *    authors.scala tag1 tag2
+ *
+ * or if on non unixy os:
+ *    scala authors.scala tag1 tag2
+ *
+ * requires scala 2.11.x+ and command line git on path
```

Comment:
Yeah, I guess, what would we gain from it though (unless we make something grander for releasing where it is a part)?